go-controller/pkg/ovn/egressip.go

```diff
@@ -891,16 +885,17 @@ func (e *egressIPDial) dial(ip net.IP) bool {
 	return true
 }
 
-func getClusterSubnets() (*net.IPNet, *net.IPNet) {
-	var v4ClusterSubnet, v6ClusterSubnet *net.IPNet
+func getClusterSubnets() ([]*net.IPNet, []*net.IPNet) {
```

Comment:
This function is kind of superfluous now; you could just have `createDefaultNoReroutePodPolicies` etc iterate `config.Default.ClusterSubnets` directly, and call `utilnet.IsIPv6CIDR` there in that loop

Reply:
`getClusterSubnets` is also called in `addNodeForEgress` and `deleteNodeForEgress` though